gradle/libs.versions.toml

```diff
@@ -23,7 +23,9 @@ googleid = "1.1.1"
 horologist = "0.6.23"
 jetbrainsCompose = "1.7.3"
 jetbrainsLifecycle = "2.8.4"
-jetbrainsNavigation = "2.8.0-alpha13"
+# if you update this, you probably need to update all other compose dependencies
+# see https://youtrack.jetbrains.com/issue/CMP-7571
+jetbrainsNavigation = "2.7.0-alpha07"
```

Comment:
@cbeyls I had to downgrade the version of navigation here or it would pull compose `1.8.0-alphas` transitively that are not compatible with `1.7`. Hope this is OK.

Reply:
Yes, I made a mistake by upgrading to a too recent version. The goal was to use the latest one compatible with Compose 1.7

Reply:
Actually the version to use should be 2.8.0-alpha12, not 13

Reply:
Ah, that makes sense 👍 I just reverted to the previous version in the git history. Feel free to change it!